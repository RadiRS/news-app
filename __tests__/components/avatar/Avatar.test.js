import 'react-native';
import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from '../../../src/components/avatar';

test('renders Avatar component correctly', () => {
  const tree = renderer.create(<Avatar photo={photo} />).toJSON();
  expect(tree).toMatchSnapshot();
});

const photo = 'http://lorempixel.com/640/480';
