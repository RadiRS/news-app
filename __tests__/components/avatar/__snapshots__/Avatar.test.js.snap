// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Avatar component correctly 1`] = `
<Image
  source={
    Object {
      "uri": "http://lorempixel.com/640/480",
    }
  }
  style={
    Array [
      Object {
        "backgroundColor": "black",
        "borderRadius": 22,
        "height": 44,
        "width": 44,
      },
    ]
  }
/>
`;
